tests/ui/intrinsics/incorrect-transmute.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0658]: intrinsics are subject to change
+error[E0658]: rust-intrinsic ABI is an implementation detail and perma-unstable
```

Comment:
nit, what about:

> the "rust-intrinsic" ABI is [...]

Reply:
and similarly for above

Reply:
Sure! should it be `extern "abi"` so it looks like what people actually write in syntax?

Reply:
pushed a diff for consideration~